_posts/2023-06-11-vanilla-os-orchid-devlog.md

```diff
@@ -0,0 +1,195 @@
+
+*Prometheus was the Titan of Greek mythology who stole fire from the gods to give it to humans.
+
+## Eratosthenes
+
+With the switch to Debian as the base for Orchid, we had to create our own repositories to manage package freezes and packages. It was not only a week-long effort, but it led to the need for a platform to display package details from the repositories.
+
+Unfortunately, most currently available platforms for this purpose did not meet our UX standards and offered a complex and outdated interface. That's why we developed [Eratosthenes*](https://github.com/Vanilla-OS/Eratosthenes), a platform based on Flask and written in Python, which allows easy searching and reading of package details directly from our repositories.
+
+You can check out the platform by visiting [repo.vanillaos.org](https://repo.vanillaos.org).
+
+*Eratosthenes was a Greek mathematician, astronomer, and geographer.
+
+## Atlas
+
+With the introduction of OCI images in Vanilla OS, there arose a need to provide a convenient and simplified method for developers to access image details. For this reason, we created [Atlas*](https://github.com/Vanilla-OS/Atlas), a platform that lists the images used in Vanilla OS and allows users to read their specifications and composition.
+
+Atlas is built with VueJS and designed to be easily navigatable, displaying all image details clearly and intuitively. Users can read about the build steps, size, layers, and other details for creating a variant of Vanilla OS.
+
+Currently, the platform showcases the Pico, Core, Desktop and Dev images, but this list might expand in the future to include new compliant images.
+
+You can test the platform by visiting [images.vanillaos.org](https://images.vanillaos.org/).
+
+*Atlas was the Titan of Greek mythology condemned to hold up the heavens or sky for eternity.
+
+## New Wallpaper
+
+Like every Linux distribution that sees a new major update, Vanilla OS Orchid has received a new official wallpaper. [Kramo](https://github.com/kra-mo) has been working hard on this. He has made a wallpaper highlighting the new light that Orchid brings to Vanilla OS.
+
+![Orchid Wallpaper, light variant: A 2D image of vanilla orchid over a sky-based wallpaper](/assets/uploads/orchid-light.webp)
+
+The new wallpaper comes in two colors, light and dark, and it represents a vanilla orchid in a pure environment, which embodies the pure essence of Vanilla OS and the desire to provide a simple and relaxing user experience.
+
+![Orchid Wallpaper, dark variant: A 2D image of vanilla orchid over a plant-based wallpaper](/assets/uploads/orchid-dark.webp)
+
+## Other news
+
+There are many new features we have worked on, and we cannot cover them all in detail in a single article for various reasons. However, we can briefly mention some of them.
+
+![Drivers Utility](/assets/uploads/drivers-utility.webp)
```

Comment:
This should be placed after the next paragraph since it looks a bit out of co text here.

Reply:
Done, updated it.